modules/preview-react/table/stories/Table.stories.mdx

```diff
@@ -0,0 +1,52 @@
+import {SymbolDoc, Specifications} from '@workday/canvas-kit-docs';
+import {Table} from '@workday/canvas-kit-preview-react/table';
+// examples
+import {Basic} from './examples/Basic';
+import {BasicWithHeading} from './examples/BasicWithHeading';
+import {FixedColumn} from './examples/FixedColumn.tsx';
+
+<Meta title="Preview/Table" component={Table} />
+
+# Canvas Kit Table
+
+`Table` is a simple styled compound component that renders a
+[table](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/table) element. It is used to
+present information in a two-dimensional table comprised of rows and columns of cells containing
+data.
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-preview-react
+```
+
+## Usage
+
+### Basic Example with Heading
```

Comment:
Is there a recommendation from a11y on using a heading vs a caption? From the [W3C](https://www.w3.org/WAI/tutorials/tables/caption-summary/):

> A caption functions like a heading for a table.

Would you ever use both a heading and a caption? I ask because it would be ideal to pick a single "Basic" example (for consistency with our other component docs) and then have the other option be the next example (either "With Heading" or "With Caption").

Reply:
Yeah definitely. 

I don't think there is a recommendation. Captions help with accessibility but, since the table itself will be accessible since it's a semantic table element, I don't think captions are absolutely necessary. Heading is probably the way most users will go